src/Core/Framework/Bundle.php

```diff
@@ -27,7 +28,7 @@ abstract class Bundle extends SymfonyBundle
     public function build(ContainerBuilder $container): void
     {
         parent::build($container);
-
+        $this->loadAdditionalClassLoaders();
```

Comment:
Is it sufficient to only call this hook when building the container? How are the classloaders registered on a normal request which does not cause the container to be rebuilt?

Reply:
I had it in mind that we used to that on container-building but you are right this is not always the case. It has to be done here at least for registering classes for service definitions. I'll check that.

Reply:
getAdditionalBundles :) there we go

Reply:
You do you need to add class loaders in the build step? We load our additional class loaders in the `getAdditionalBundles` and it works pretty well.

Reply:
A service definition can contain a class from the plugins dependencies. It is not ensured that on every plugin action the bundles have been checked (AFAIK installation is not loading it). This way we have to add it as well to the container building.